test/e2e/framework/framework.go

```diff
@@ -58,7 +59,12 @@ const (
 type Framework struct {
 	BaseName string
 
-	Client        *client.Client
+	// Client is manually created and should not be used unless absolutely necessary.  Use Clientset_1_5
+	// where possible.
+	Client *client.Client
+	// ClientSet uses internal objects, you should use Clientset_1_5 where possible.
```

Comment:
nit: change "ClientSet" to "Clientset"? Or change "Clientset_1_5" to "ClientSet_1_5". If the change requires more than grep+sed then I can live with the mismatch.